config/flags_network.go

```diff
@@ -118,7 +118,13 @@ var (
 	FlagTraversal = cli.StringFlag{
 		Name:  "traversal",
 		Usage: "Comma separated order of NAT traversal methods to be used for providing service",
-		Value: "holepunching,upnp,manual",
+		Value: "manual,upnp,holepunching",
+	}
+	// FlagPortCheckServers list of asymmetric UDP echo servers for checking port availability
+	FlagPortCheckServers = cli.StringFlag{
```

Comment:
make this hidden, users will have little clue as to what it is and are unlikely to use this, ever. This just adds another flag to an already confusing list :)